modules/webform_event_dispatcher/src/Event/WebformElement/WebformElementTypeAlterEvent.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Drupal\webform_event_dispatcher\Event\WebformElement;
+
+/**
+ * Class WebformElementTypeAlterEvent.
+ *
+ * @package Drupal\webform_event_dispatcher\Event\Element
+ */
+class WebformElementTypeAlterEvent extends WebformElementAlterEvent {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getDispatcherType() {
+    $type = $this->getElement()['#type'];
```

Comment:
Maybe add this as a new function to the WebformElementAlterEvent? `getType` or something?

Reply:
Good call